src/player.cpp

```diff
@@ -259,6 +259,9 @@ void Player::MainLoop() {
 
 	auto frame_limit = DisplayUi->GetFrameLimit();
 	if (frame_limit == Game_Clock::duration()) {
+#ifdef EMSCRIPTEN
+		emscripten_sleep(0);
```

Comment:
this was the reason why my other ASYNC PR never worked: I forgot to add a sleep to this return here. (this sleep 0 is "yield to browser" otherwise nothing will render)